components/org.wso2.carbon.identity.api.server.configs/org.wso2.carbon.identity.api.server.configs.v1/src/gen/java/org/wso2/carbon/identity/api/server/configs/v1/model/CORSConfig.java

```diff
@@ -110,6 +131,25 @@ public CORSConfig addSupportedMethodsItem(String supportedMethodsItem) {
     }
 
         /**
+    * If true the CORS valve must support any requested header, else the supported headers list must be consulted.
+    **/
+    public CORSConfig supportAnyHeader(Boolean supportAnyHeader) {
+
+        this.supportAnyHeader = supportAnyHeader;
+        return this;
+    }
+    
+    @ApiModelProperty(example = "false", value = "If true the CORS valve must support any requested header, else the supported headers list must be consulted.")
```

Comment:
Lines better be trimmed at 120 column for better readability

Reply:
@ruwanta This is generated code by `openapi-generator-maven-plugin`.